src/card.css

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  align-items: center;
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  color: #060b35;
+  margin-top: 17px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 23px;
+}
+
+.card__price-title {
+  color: #616070;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
```

you can add border with transparent color by default and change border color on hover 😉

```suggestion
.card__button {
  border: 1px solid transparent;
  box-sizing: border-box;
```